include/mbedtls/platform.h

```diff
@@ -164,9 +164,10 @@ int mbedtls_platform_set_fprintf( int (*fprintf_func)( FILE *stream, const char
 extern int (*mbedtls_printf)( const char *format, ... );
 
 /**
- * \brief    This function sets your own printf function pointer.
+ * \brief    This function allows configuring a custom \c fprintf function 
```

Comment:
Blocker: This should be `printf` instead of `fprintf`.

Reply:
Fixed